### ISSUE TITLE
Uncaught Error: Could not locate the bindings file.

### ISSUE THREAD

**Issue by tylerdcooper** [j03b/language-circuitpython#2]
Getting the following error when trying to install atom-circuitpython. Any ideas?

node -v v12.13.0
node-gyp -v v6.0.1

**Atom**: 1.41.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.14.6
**Thrown From**: [language-circuitpython](https://github.com/jos-b/language-circuitpython) package 0.6.0

### Stack Trace

Uncaught Error: Could not locate the bindings file. Tried:
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/out/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/out/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/default/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/compiled/10.11.0/darwin/x64/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/release/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/debug/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/default/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/lib/binding/node-v69-darwin-x64/bindings.node

```
At /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/bindings.js:135

Error: Could not locate the bindings file. Tried:
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/out/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/Debug/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/out/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/Release/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/build/default/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/compiled/10.11.0/darwin/x64/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/release/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/debug/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/addon-build/default/install-root/bindings.node
 → /Users/tylercooper/.atom/packages/language-circuitpython/node_modules/bindings/lib/binding/node-v69-darwin-x64/bindings.node
    at bindings (/packages/language-circuitpython/node_modules/bindings/bindings.js:126:9)
    at /packages/language-circuitpython/node_modules/@serialport/bindings/lib/darwin.js:1:191)
    at /packages/language-circuitpython/node_modules/@serialport/bindings/lib/darwin.js:104:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146745)
    at /packages/language-circuitpython/node_modules/@serialport/bindings/lib/index.js:10:22)
    at /packages/language-circuitpython/node_modules/@serialport/bindings/lib/index.js:17:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146745)
    at /packages/language-circuitpython/node_modules/serialport/lib/index.js:2:17)
    at /packages/language-circuitpython/node_modules/serialport/lib/index.js:17:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146745)
    at /packages/atom-circuitpython/lib/board-view.js:5:45)
    at /packages/atom-circuitpython/lib/board-view.js:14:23)
```

### Commands

```
     -0:17.5.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```
atom-circuitpython 0.5.1 
fsync-on-save 1.0.4 
language-circuitpython 0.6.0 
```

**Comment by jos-b** [comment]
It's an error to do with the compiled sections of the serialport module, used for detection of serialports which circuitpython devices identify as. I'm not sure how to fix it myself though serialport/node-serialport#847 may be of interest.

**Comment by j03b** [comment]
I'll hastily say this is fixed with 0.7.1, but if this resurfaces feel free to reopen it again
